grunt_tasks/webpack.config.js

```diff
@@ -106,7 +106,7 @@ module.exports = {
             // JSON files
             {
                 test: /\.json$/,
-                include: loaderPaths,
+                include: [...loaderPaths, ...loaderPathsNodeModules],
```

Comment:
Did we ever decide on a minimum node version for development?  This requires v4, I think.